docs/guide/multisite.md

```diff
@@ -2,109 +2,201 @@
+The two configuration parameters, `location_id` and `excluded_uri_prefixes` are taken into account in several places,
+for example in URL alias lookups for incoming web requests, and URL alias path generation in Twig templates.
+In addition you need to consider them when generating site paths in your layout, or other places rendering site/tree structure.
 
-##### `content.tree_root.excluded_uri_prefixes`
+#### `location_id`
 
-Use this parameter to exclude a list of URIs from the root jail defined using `location_id` described above. Add as many locations pathString as you like. In the example above, the Media and Images folders will be accessible using their own URI even though they're not below `tree_root.location_id`.
+`content.tree_root.location_id` sets the Location ID of the content root of your site
+and nothing above this level will be accessible by default.
+This parameter can be filtered using the `excluded_uri_prefixes` parameter.
 
-!!! note
+#### `excluded_uri_prefixes`
 
-    Leading slashes (`/`) are automatically trimmed internally by eZ Platform, so they can be ignored.
+`content.tree_root.excluded_uri_prefixes` defines which URIs ignore the root limit set using `location_id`.
+Add as many Location pathStrings as you like.
+In the example above, the Media and Images folders will be accessible using their own URI even though they are not below `content.tree_root.location_id`.
 
-## Usage
+!!! note
 
-### Multisite Design
+    Leading slashes (`/`) are automatically trimmed internally, so they can be ignored.
 
-eZ Platform does not apply a [Legacy template fallback](https://doc.ez.no/display/EZP/Legacy+template+fallback) like eZ Publish did. You can, however, have different designs in your multisite installation if you organize the view configuration with the use of siteaccesses.
+## Configuration example
 
-You can see how this is done on the example of a multisite that contains two blogs with different designs, but sharing some templates. Let's assume you have a siteaccess called `second_site`. You can organize your templates in the following folder structure:
+To see how multisite can be used, let's look at an example of two sites using the same eZ Platform instance: a general site and a children's site.
+Separate SiteAccess groups are set up for the two sites:
 
 ``` yaml
-views
- |- pagelayout.html.twig
-     |- full
-         |- blog.html.twig
-         |- blog_post.html.twig
-     |- second_site
-         |- pagelayout.html.twig
-         |- full
-             |- blog_post.html.twig
+ezpublish:
+    siteaccess:
+        list: [site, kids]
+        groups:
+            site_group: [site]
+            kids_group: [kids]
+        default_siteaccess: site
+        match:
+            URIElement: 1
 ```
 
-Now you can use this view configuration (stored e.g. in a `views.yml` file):
+This is your content structure:
+
+```
+├── General
+│   └── Articles
+│       └── Mammals
+├── Children
```

Comment:
Hmm, the visualization here seems to not include location 2, often called "Content", it would typically be the parent of the site roots, meaning they will both pop up in  content section of backend UI.